File: src/main/java/no/novari/flyt/egrunnerverv/gateway/instance/model/EgrunnervervSakKlassering.java
package no.novari.flyt.egrunnerverv.gateway.instance.model;

import jakarta.validation.constraints.NotNull;
import lombok.Builder;
import lombok.EqualsAndHashCode;
import lombok.Getter;
import lombok.extern.jackson.Jacksonized;

@Getter
@Jacksonized
@EqualsAndHashCode
@Builder
public class EgrunnervervSakKlassering {
    @NotNull
    private final String ordningsprinsipp;
    @NotNull
    private final String ordningsverdi;
    @NotNull
    private final String beskrivelse;
    @NotNull
    private final String sortering;
    @NotNull
    private final String untattOffentlighet;
}

File: src/main/kotlin/no/novari/flyt/egrunnerverv/gateway/instance/model/EgrunnervervSakKlassering.kt
package no.novari.flyt.egrunnerverv.gateway.instance.model

data class EgrunnervervSakKlassering(
    val ordningsprinsipp: String,
    val ordningsverdi: String,
    val beskrivelse: String,
    val sortering: String,
    val untattOffentlighet: String,
)
